stats/memtier

```diff
@@ -98,33 +99,16 @@ while (line = $stdin.gets)
 end
 
 def gen_output_sum(type, histo_sum)
-  puts "total_#{type}_ops/s: #{histo_sum[1]}"
-  puts "total_#{type}_hits/s: #{histo_sum[2]}"
-  puts "total_#{type}_misses/s: #{histo_sum[3]}"
-  puts "total_#{type}_avg_latency_ms: #{histo_sum[4]}"
-  puts "total_#{type}_p50_latency_ms: #{histo_sum[5]}"
-  puts "total_#{type}_p99_latency_ms: #{histo_sum[6]}"
-  puts "total_#{type}_p99.9_latency_ms: #{histo_sum[7]}"
-  puts "total_#{type}_kb/s: #{histo_sum[8]}"
+  (1..UNITS.size - 1).each do |i|
```

Comment:
thanks, just help change to (1...UNITS.size) which excludes UNIT.size directly. Similar comments to other parts.